diff --git a/dist/cli.js b/dist/cli.js
index 5672733813ad0999e4f12d05d751951bc9a472a5..dd92b95369baa5fb67cfb019544dbbce14a450a1 100644
--- a/dist/cli.js
+++ b/dist/cli.js
@@ -237,9 +237,6 @@ var customBabel = (() => {
         }, {
           name: '@babel/plugin-proposal-class-properties',
           loose: true
-        }, !customOptions.modern && {
-          name: '@babel/plugin-transform-regenerator',
-          async: false
         }, {
           name: 'babel-plugin-macros'
         }].filter(Boolean));
diff --git a/dist/microbundle.js b/dist/microbundle.js
index 91fd66a0e8da2f8e961c0f6fd43f00a926f0fc45..7b6bc7cb53688b5660156b1cd8b5ba65020b3bdf 100644
--- a/dist/microbundle.js
+++ b/dist/microbundle.js
@@ -236,9 +236,6 @@ var customBabel = (() => {
         }, {
           name: '@babel/plugin-proposal-class-properties',
           loose: true
-        }, !customOptions.modern && {
-          name: '@babel/plugin-transform-regenerator',
-          async: false
         }, {
           name: 'babel-plugin-macros'
         }].filter(Boolean));
